TESTS/spm/neg_server_tests/neg_tests.h

```diff
@@ -0,0 +1,32 @@
+/* Copyright (c) 2017 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+ 
+#define PSA_NEG_TEST_NAME(test_name) psa_neg_test_ ## test_name
+
+#define PSA_NEG_TEST(test_name)       \
```

Comment:
This is a strange way to write a test. Is the idea that there are an unknown amount of tests?

Reply:
This is kind of our own framework for negative testing